2022/40xxx/CVE-2022-40621.json

```diff
@@ -1,18 +1,82 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2022-40621",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.9"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2022-40621",
+    "ASSIGNER": "cve@rapid7.com",
+    "DATE_PUBLIC": "2022-07-05T14:00:00.000Z",
+    "TITLE": "WAVLINK Quantum D4G (WN531G3) Pass-The-Hash",
```

Comment:
Their website shows the casing as `Wavlink` rather than all capital letters. See other fields (`vendor_name` and `description_data`).

Reply:
My source of truth on caps is:

https://www.wavlink.com/en_us/about/Company_Overview.html

> Headquartered in Shenzhen and with over 15 years of technology expertise, WAVLINK brand is rising rapidly in market

Reply:
Basically, I always look at things like About and Privacy Policies and TOSes to figure out how their lawyers spell the company name